packages/theme-language-server-common/src/hover/providers/LiquidTagHoverProvider.spec.ts

```diff
@@ -18,16 +18,17 @@ describe('Module: LiquidTagHoverProvider', async () => {
   });
 
   it('should return the hover description of the correct tag', async () => {
+    // cursor always points at character before █
     await expect(provider).to.hover(`{%█ if cond %}{% endif %}`, expect.stringContaining('if'));
     await expect(provider).to.hover(`{% i█f cond %}{% endif %}`, expect.stringContaining('if'));
     await expect(provider).to.hover(`{% if█ cond %}{% endif %}`, expect.stringContaining('if'));
+    await expect(provider).to.hover(`{% if █cond %}{% endif %}`, expect.stringContaining('if'));
     await expect(provider).to.hover(`{% if cond █%}{% endif %}`, expect.stringContaining('if'));
     await expect(provider).to.hover(`{% if cond %}{% █ endif %}`, expect.stringContaining('if'));
     await expect(provider).to.hover(`{% echo█ 'hi' %}`, expect.stringContaining('echo'));
   });
 
   it('should not return the tag hover description when hovering over anything else in the tag', async () => {
-    await expect(provider).to.not.hover(`{% if █cond %}{% endif %}`, expect.stringContaining('if'));
```

Comment:
The cursor points to the the thing _left_ of the cursor, so you're not technically _in cond_ when your cursor is before the `c`.